File: JobMtaani.Web/app/models/adDetailsModel.ts
module app.models {
    export interface IAdDetailsModel {
        AdDetails: app.domain.IAd;
        AdApplicantDetails: app.domain.ProfileModel[];
    }

    export class AdDetailsModel implements IAdDetailsModel {
        constructor(public AdDetails: app.domain.IAd, public AdApplicantDetails: app.domain.ProfileModel[]) {
        }
    }
}

File: JobMtaani.Web/Models/AdDetailsModel.cs
using JobMtaani.Business.Entities;
using System;
using System.Collections.Generic;
using System.Linq;
using System.Web;

namespace JobMtaani.Web.Models
{
    public class AdDetailsModel
    {
        public Ad AdDetails { get; set; }
        public List<UserAccountModel> AdApplicantDetails { get; set; }

        public AdDetailsModel()
        {
            AdDetails = new Ad();
            AdApplicantDetails = new List<UserAccountModel>();
        }
    }

    public class NewsFeedModel
    {
        public Ad AdDetails { get; set; }
        public AdApplication AdApplication { get; set; }
    }
}